.github/workflows/validate_new_quickstarts_configuration.yaml

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright 2022 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Validate Configuration API
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - feature/new_quickstarts
+      - release-*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - feature/new_quickstarts
+      - release-*
+jobs:
+  validate:
+    name: Validate quickstart for `${{ matrix.quickstart_language }}` with `${{ matrix.quickstart_variant }}` on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    env:
+      DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/master/install
+      GOVER: 1.17
+      KUBERNETES_VERSION: v1.21.1
+      KIND_VERSION: v0.11.0
+      KIND_IMAGE_SHA: sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-10.15]
```

Comment:
ideally if possible change macOS.
Can be a separate issue to track the change also. cc @dapr/maintainers-quickstarts